README.md

```diff
@@ -229,6 +229,13 @@ A table of errors thrown by the parser:
 |`err.code`|`err.args`|Description|
 |----------|----------|-----------|
 |`invalidCheckboxDefault`||Expect 0 or 1.|
+|`invalidRange`||Expect a number or an array.|
```

Comment:
The version in Stylus won't accept a plain number, only an array. Will this parser eventually replace the one in Stylus then? If so, then being able to set only the default value is good; if not, then this should say "Expect an array".

Reply:
We can keep this as is. BTW :+1: for the [reminder](https://github.com/StylishThemes/parse-usercss/issues/27#issuecomment-423521514)

Reply:
To clarify, I think we can accept a simple number so instead of:
```
@var number my-number "My number [10]
```
The author can write
```
@var number my-number "My number 10
```

Reply:
The examples you shared are missing the closing double quote, but I'm guessing that's just a typo.